### PR TITLE
Create SQL commands intro workshop

### PR DESCRIPTION
**Pull request by macintoshhelper**
Has 7 questions/tasks currently.
Has some links for accompanied resources.

Not sure if it's long enough yet?

https://github.com/foundersandcoders/master-reference/issues/291
#2 #3 #4

### CODE REVIEW

**Comment by jsms90** [comment]
@macintoshhelper This seems like a recreation of the [existing exercises](https://pgexercises.com/questions/basic/) that students are pointed to?

**Comment by macintoshhelper** [comment]
@jsms90 The data is more "real world" - I find it's harder to learn SQL if using more abstract data. I tried to cover similar content, since I feel it progresses nicely.

File: README.md [hunk 1]
```diff
@@ -1 +1,143 @@
...
+
+## Syntaxes
+psql CLI
+```
+\c <database name> # Connect to database
+\i <file path> # Load SQL build file into currently open database
+```
+
+## Getting Started
+Make sure you have installed and setup PostgreSQL.
```
Comment (shiryz) [inline README.md]: Add a link to installation instructions, [DWYL's learn ppostgres](https://github.com/dwyl/learn-postgresql)

**Review by shiryz — CHANGES_REQUESTED** [review]
Looks really good, one small change

File: README.md (hunk 1 above)
Comment (macintoshhelper) [reply README.md]: I've added a link to my installation instructions (proposed change in `master-reference`).

**Review by shiryz — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,173 @@
-# sql-commands-intro
\ No newline at end of file
+# SQL Commands Introduction
+This workshop will teach you how to use basic SQL - Structured Query Language. SQL is a standard for relational database systems, while MySQL and PostgreSQL are implementations of the standard.
+
+## Syntaxes
```
Comment (jsms90) [inline README.md]: This section seems a bit redundant. You have the same information in  "Setting up the workshop database"

**Review by jsms90 — CHANGES_REQUESTED** [review]
Minor thing.

Looks great @macintoshhelper

**Review by shiryz — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (macintoshhelper) [reply README.md]: Replaced it with workshop-specific info.

Comment (jsms90) [reply README.md]: Sorry, this comment refers to the whole "Syntaxes" section as duplicate info of what's in the "Setting up the workshop database" section

Comment (macintoshhelper) [reply README.md]: Ok, I've removed syntaxes

**Review by jsms90 — APPROVED** [review]

**Pull request merged**